lib/src/screens/exchange/exchange_page.dart

```diff
@@ -456,9 +456,13 @@ class ExchangePage extends BasePage {
     depositAmountController.addListener(() {
       if (depositAmountController.text != exchangeViewModel.depositAmount) {
         _depositAmountDebounce.run(() { 
-          exchangeViewModel.changeDepositAmount(
-              amount: depositAmountController.text);
-          exchangeViewModel.isReceiveAmountEntered = false;
+      if (exchangeViewModel.checkIfInputMeetsMinOrMaxCondition(
+                   depositAmountController.text) ==
+               true){
+  exchangeViewModel.changeDepositAmount(
+      amount: depositAmountController.text);
+  exchangeViewModel.isReceiveAmountEntered = false;
+}
```

Comment:
- format
- duplicate boolean check (i.e bool == true)

Reply:
Remove the duplicated boolean and fixed the formatting